tar-unix.opam

```diff
@@ -8,20 +8,18 @@ dev-repo:     "https://github.com/mirage/ocaml-tar.git"
 doc:          "https://mirage.github.io/ocaml-tar/"
 
 build: [
-  [ "jbuilder" "build" "--only-packages=tar,tar-unix" ]
-]
-
-build-test: [
-  [ "jbuilder" "runtest" ]
+  ["jbuilder" "subst" "-p" name "--name" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
 ]
 
 depends: [
-  "jbuilder"          {build}
+  "jbuilder"          {build & >="1.0+beta7"}
```

Comment:
I likely have a stupid question, but do the other opam files not require a `jbuilder >= "1.0+beta7"`?

Reply:
I don't think that's a stupid question :) I resync'ed the `jbuild` files from `opam-repository` and only `tar-unix` seems to have that. I bet they should all have the `>= 1.0+beta7` really-- I'll adjust that.